_config.yml

```diff
@@ -54,6 +58,30 @@ markdown: kramdown
 kramdown:
   input: GFM
 
+# Google Analytics
+google-analytics:
+  enabled: false
+  code: "UA-XXXXXX-1"
+
+# US Digital Analytics Program - https://analytics.usa.gov
+digital-analytics-program:
+  enabled: false
+  agency: ""
+  subagency: ""
+
+# Hypothesis.io - Annotator
+hypothesis:
+  enabled: true
+
+# Twitter
+twittercard:
+  enabled: false
+  site: ""
```

Comment:
Oh! I think `username` would be the most obvious property name. But it looks like `addons/twittercard.html` doesn't use this property either way.